system/.path.macos

```diff
@@ -1,2 +1,2 @@
 prepend-path "/usr/local/opt/coreutils/libexec/gnubin"
-prepend-path "/usr/local/opt/ruby/bin"
+prepend-path "/usr/local/opt/ruby/bin"
```

Comment:
Please leave the newline here, no whitespace changes needed.

Reply:
OK. Fix it.